README.md

```diff
@@ -3,6 +3,34 @@ Fast, adaptive, secure, and timeless design network architecture.
 
 A design architecture for the web that enables real-time updating of user experiences based on changing inputs.
 
+## Packages
+### FAST animation library (Beta)
```

Comment:
@Falkicon how do we feel about removing the "(Beta)" tags on our packages?  The package version itself could still include, but, just not sure it's necessary to be documented as Beta.

Reply:
Executive decision, remove.  :-)  We can add it back later if necessary.

Reply:
Roger -updating.